src/main/java/frc/robot/commands/ColorWheelWithJoystick.java

```diff
@@ -0,0 +1,123 @@
+    ColorMatchResult match = colorMatcher.matchClosestColor(detectedColor);
+    String desiredColor;
+    String gameSpecificColor = DriverStation.getInstance().getGameSpecificMessage();
+    Elevator elevator;
+  
+    public ColorWheelWithJoystick() {
+    }
+
+    public void initialize() {
+      switch (gameSpecificColor){
+        case "R":
+       wheelColor = Elevator.WheelColor.BLUE;
+        case "B":
+        wheelColor = Elevator.WheelColor.RED;
+        case "G":
+        wheelColor = Elevator.WheelColor.YELLOW;
+        case "Y":
+        wheelColor = Elevator.WheelColor.GREEN;
+      }
+
+        /*if (gameSpecificColor != null){
+        colorMatcher.addColorMatch(kBlueTarget);
+        colorMatcher.addColorMatch(kGreenTarget);
+        colorMatcher.addColorMatch(kRedTarget);
+        colorMatcher.addColorMatch(kYellowTarget);
+        //Color detectedColor = colorSensor.getColor();
+        //ColorMatchResult match = colorMatcher.matchClosestColor(detectedColor);
+
+          switch (gameSpecificColor)
+          {
+            case "B" :
+              //Blue case code
+              desiredColor = "R";
+              pwm.setSpeed(Const.Speed.COLOR_WHEEL_FIXED_SPEED); //either negative or postive for ccw or cw
+              break;
+            case "G" :
+              //Green case code
+              desiredColor = "Y";
+              pwm.setSpeed(Const.Speed.COLOR_WHEEL_FIXED_SPEED);
+         
+              break;
+            case "R" :
+              //Red case code
+              desiredColor = "B";
+              pwm.setSpeed(Const.Speed.COLOR_WHEEL_FIXED_SPEED);
+                break;
+            
+            case "Y" :
+              //Yellow case code
+              desiredColor = "G";
+              pwm.setSpeed(Const.Speed.COLOR_WHEEL_FIXED_SPEED);
+                break;
+            
+            default :
+              //This is corrupt data
+              pwm.setSpeed(0.0d);
+              break;
+          }
+        } else {
+          //Code for no data received yet
+          pwm.setSpeed(0.0d);
+        }*/
+      }
+
+    public void execute() {
+
+        if (desiredColor == null || rotationComplete == false){
+            elevator.rotateColorWheel(3);
+            rotationComplete = true;
+        } else {
+          elevator.goToColor(wheelColor);
+          colorFound = true;
+        }
+
+        /** Print values to the Smart Dashboard */
+        SmartDashboard.putNumber("ColorSensor2/Red", colorSensor.getRed());
+        SmartDashboard.putNumber("ColorSensor2/Green", colorSensor.getGreen());
+        SmartDashboard.putNumber("ColorSensor2/Blue", colorSensor.getBlue());
+        SmartDashboard.putString("ColorSensor2/Color", colorString);
+        SmartDashboard.putString("ColorSensor2/desiredColor", desiredColor);
+        SmartDashboard.putBoolean("ColorSensor2/ColorFound", colorFound);
+        SmartDashboard.putBoolean("ColorSensor2/ ConditionMet", colorString.equals(desiredColor));
```

Comment:
These will only be updated as long as the command is running. This is not recommended because then as soon as the command ends, we've got a bunch of stale entries on the dashboard.